src/app/actions.js

```diff
@@ -205,3 +210,49 @@ export function connectAndStartPolling(reduxStore, bakery) {
   connectAndStartPolling.NAME = "connectAndStartPolling";
   return connectAndStartPolling;
 }
+
+export async function connectAndListModels(
```

Comment:
This was moved from `[src/app/model-poller.js](https://github.com/canonical/jaas-dashboard/pull/1343/files#diff-70c5a8c9509f3d19534d6056094d41f47d6ae1f9a3b565679551acd7b42d0dd1)` without any changes.